CityPuzzle/Login And SingUp/SignUpPage.xaml.cs

```diff
@@ -73,7 +73,7 @@ void Registration_Click(object sender, EventArgs e)
                 {
                     user = CreateUser(name: nameEntry.Text, userName: usernameEntry.Text, lastName: lastnameEntry.Text, email: emailEntry.Text, password: User.PassToHash(passEntry.Text));
                 }
-                Sql.SaveUser(user);
+                user=Sql.SaveUser(user);//nezinau ar sitas tinka- man norisi perkleti i useri ir naudoti kaip non-static metoda.
```

Comment:
This is unusable necause user is a local variable and can't be accessed outside this method. SaveUser should not return anything